lib/solidus_affirm/factories.rb

```diff
@@ -1,6 +1,5 @@
-FactoryGirl.define do
-  # Define your Spree extensions Factories within this file to enable applications, and other extensions to use and override them.
-  #
-  # Example adding this to your spec_helper will load these Factories for use:
-  # require 'solidus_affirm/factories'
+require 'factory_girl'
+
+Dir["#{File.dirname(__FILE__)}/factories/**"].each do |f|
```

Comment:
Doesn't factory girl load factories automatically if they're defined in a way it expects? I've never had to manually require them.